machAeroTutorials/aero_gridRefinementStudy.rst

```diff
@@ -66,8 +66,114 @@ An example of grid convergence plot for a family of RAE 2822 Airfoil meshes is i
 
     Figure 1: Grid convergence plot for RAE 2822 Transonic Airfoil.
 
+
+
+2. Grid Refinement Study on Airfoils
+======================================
+
+Instead of using the :code:`cgns_utils coarsen` feature, we can easily make the finer or coarsen meshes with the help of  :code:`prefoil` package.
+The main reason behind this idea is to generate the meshes without changing the :code:`growth rate` of the off wall layers.
+If you use :code:`cgns_utils coarsen` feature, you will be able to increase the first off wall spacing  :code:`s0` uniformly; however, the grow ratio is going to change and the off wall layers will have too much distance between each other.
+
+In order to avoid this, we can use the :code:`prefoil` packages easily and still be able to coarsen or refine the meshes. 
+The example code is given below. You can either upload a :code:`.dat` file or create the NACA 4 digit airfoils. 
+Then, you can manipulate the meshing parameters and get mesh grids with different levels.
+
+.. code-block:: python
+        
+    from pyhyp import pyHyp
```

Comment:
Python runscripts should not go into the RST file, but into a separate python file. That way, it can be run by users and can be regression tested.